script.js

```diff
@@ -0,0 +1,72 @@
+   
+              //Кнопка proMod
+let onOf = true;
+let proModOnOffButton = document.getElementById('proMod');
+let proModDiv = document.getElementById('addKeyRow');
+proModOnOffButton.addEventListener("click", visOnOff);
+
+function visOnOff() {
+    if(onOf === true){
+        proModDiv.style.visibility = 'visible';

```

Comment:
Could we create a function which will by expect 3 arguments: `visibility, backgroundColor, textColor`